### PR TITLE
added dozenodes

### PR DESCRIPTION
**Pull request by ddozen**

### CODE REVIEW

**Comment by ddozen** [comment]
https://kusama.subscan.io/extrinsic/17141029-3

**Comment by stepanLav** [comment]
Thank you for your request!
@ddozen   

You can check how it will look like:

<details>
  <summary>App screens</summary>

Screenshot 2023-03-22 at 10 00 20
Screenshot 2023-03-22 at 10 00 25
Screenshot 2023-03-22 at 10 00 29


</details>

File: registry/kusama.json [hunk 1]
```diff
@@ -110,5 +110,13 @@
...
          "isOrganization": false
-     }
+     },
+     {
+        "address": "JA9TjkzZzsJnBC8igbnLZ2WYvgsYHd6fu54QJAViqnADoZq",
+        "name": "DOZENODES",
+        "image": "https://raw.githubusercontent.com/nova-wallet/opengov-delegate-registry/master/images/dozenodes.png",
+        "shortDescription": "Kusama Validator Operator www.dozendoes.com",
+        "longDescription": "## About me\nMy name is Marcin and I'm the main technical guy behind dozenodes.com.\nI'm an active memeber of communities for multiple project in dotsama ecosytem\n I have vast technical expirience which allows me to properly assess proposed changes.\n Feel free to contact me via contact section of dozenodes.com.",
```
Comment (stepanLav) [inline registry/kusama.json:119]:
```suggestion
        "longDescription": "## About me\n\nMy name is Marcin and I'm the main technical guy behind dozenodes.com.\n\nI'm an active memeber of communities for multiple project in dotsama ecosytem\n\n I have vast technical expirience which allows me to properly assess proposed changes.\n\n Feel free to contact me via contact section of dozenodes.com.",
```

Comment (stepanLav) [reply registry/kusama.json:119]: In order to make a new line, we need a double '\n' character

**Review by stepanLav — APPROVED** [review]

**Comment by ddozen** [comment]
@stepanLav 
Thanks.
Looks good to me now.

**Pull request merged**